language.go

```diff
@@ -490,6 +491,7 @@ func NewDefinedLanguages() *DefinedLanguages {
 			"JAI":                 NewLanguage("JAI", []string{"//"}, [][]string{{"/*", "*/"}}),
 			"Janet":               NewLanguage("Janet", []string{"#"}, [][]string{{"", ""}}),
 			"Java":                NewLanguage("Java", []string{"//"}, [][]string{{"/*", "*/"}}),
+			"Jsp":                 NewLanguage("Jsp", []string{"//"}, [][]string{{"/*", "*/"}}),
```

Comment:
Can you change the `Jsp` to an upper case?
```suggestion
			"JSP":                 NewLanguage("JSP", []string{"//"}, [][]string{{"/*", "*/"}}),
```